### PR TITLE
Update Safety to draw actual speed restrictions

### PR DESCRIPTION
**Pull request by kiryldz**

### CODE REVIEW

File: Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt [hunk 1]
```diff
@@ -72,6 +77,8 @@ class SafetyActivityKt : BaseActivity() {
 
         override fun onRoadRestrictionsUpdated(roadRestrictions: RoadRestrictions) {
             maxAllowedSpeed = roadRestrictions.speedLimits.car.max
+            // set speed value to view to show it if overspeeding occurs
```
Comment (yunikkk) [inline Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt]: `set speed value to view to show if overspeeding occurs` ?

File: Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt [hunk 2]
```diff
@@ -45,16 +46,20 @@ class SafetyActivityKt : BaseActivity() {
...
             val mySpeed = vehicleState.speed
 
-            // display toast with overspeed warning if our speed is greater than maximum allowed speed
             if (mySpeed > maxAllowedSpeed && maxAllowedSpeed > 0) {
                 // all VisionListener callbacks are executed on a background thread. Need switch to a main thread
                 runOnUiThread {
+                    // show current speed limits in view
+                    speed_alert_view.visibility = View.VISIBLE
+                    // notify user by Toast that he's overspeeding
                     Toast.makeText(
```
Comment (yunikkk) [inline Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt]: Or - remove toast completely? I dont think we need two separate indications. Also - iOS does not show toasts as well

File: Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt (hunk 1 above)
Comment (kiryldz) [reply Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt]: OK

File: Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt (hunk 2 above)
Comment (kiryldz) [reply Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt]: Will remove toast and show 2 types of speed alert: normal and overspeeding

File: Examples/src/main/java/com/mapbox/vision/examples/SafetyActivity.java [hunk 3]
```diff
@@ -26,9 +31,25 @@
...
+        private int textColorId;
+
+        SpeedLimit(final int imageResId, final int textColorId) {
+            this.imageResId = imageResId;
+            this.textColorId = textColorId;
+        }
+    }
+
+    private final SpeedLimit overspeeding = new SpeedLimit(R.drawable.speed_limit_overspeeding, android.R.color.white);
+    private final SpeedLimit normal = new SpeedLimit(R.drawable.speed_limit_normal, android.R.color.black);
```
Comment (yunikkk) [inline Examples/src/main/java/com/mapbox/vision/examples/SafetyActivity.java]: Is the `speed_limit_overspeeding` icon in this PR as well? Dont see it in the diff...

Comment (kiryldz) [reply Examples/src/main/java/com/mapbox/vision/examples/SafetyActivity.java]: Indeed, git did not add it because of renaming

**Pull request merged**